beacon-chain/powchain/service.go

```diff
@@ -355,9 +355,6 @@ func (w *Web3Service) ProcessChainStartLog(depositLog gethTypes.Log) {
 	}
 
 	timestamp := binary.LittleEndian.Uint64(timestampData)
-	if uint64(time.Now().Unix()) < timestamp {
-		log.Errorf("Invalid timestamp from log expected %d > %d", time.Now().Unix(), timestamp)
```

Comment:
Can this be a warning? It would be nice or helpful to log "by the way, the chain start is in the future". 
While this is a normal event (not error level), it could still be helpful to communicate.

What do you think?

Reply:
It will never happen though, that's the thing

Reply:
Not even in development, no matter what, that condition will always trigger. For example, if time.Now() is > timestamp, that would be bad as that would mean the chain will have already started and we will be skipping a few validator slots.